examples/getting_started.py

```diff
@@ -1,34 +1,43 @@
+# Disable pylint errors for logging basicConfig
+# pylint: disable=no-logging-basicconfig
+import logging
+
+from typing import Optional
+
 from haystack.document_stores import InMemoryDocumentStore
 from haystack.utils import build_pipeline, add_example_data, print_answers
 
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```
Let's use this logger instance for debug logging.